Library/Formula/hbase.rb

```diff
@@ -4,23 +4,81 @@ class Hbase < Formula
   url "https://www.apache.org/dyn/closer.cgi?path=hbase/1.1.2/hbase-1.1.2-bin.tar.gz"
   sha256 "8ca5bf0203cef86b4a0acbba89afcd5977488ebc73eec097e93c592b16f8bede"
 
-  bottle :unneeded
-
   depends_on :java => "1.6+"
-  depends_on "hadoop"
+  depends_on "hadoop" => :optional
+  depends_on "lzo" => :recommended
+  depends_on "ant" => :build if build.with? "lzo"
+
+  resource "hadoop-lzo" do
+    url "https://github.com/cloudera/hadoop-lzo/archive/0.4.14.tar.gz"
+    sha256 "aa8ddbb8b3f9e1c4b8cc3523486acdb7841cd97c002a9f2959c5b320c7bb0e6c"
+  end
+
+  def rundir
+    var/"run/hbase"
+  end
 
   def install
     rm_f Dir["bin/*.cmd", "conf/*.cmd"]
     libexec.install %w[bin conf docs lib hbase-webapps]
     bin.write_exec_script Dir["#{libexec}/bin/*"]
 
-    inreplace "#{libexec}/conf/hbase-env.sh",
-      "# export JAVA_HOME=/usr/java/jdk1.6.0/",
-      "export JAVA_HOME=\"$(/usr/libexec/java_home)\""
+    resource("hadoop-lzo").stage do
+      # Fixed upstream: https://github.com/cloudera/hadoop-lzo/blob/master/build.xml#L235
+      inreplace "build.xml",
+                %r{(<class name="com.hadoop.compression.lzo.LzoDecompressor" />)},
+                "\\1\n<classpath refid=\"classpath\"/>"
+      Kernel.system({ "CLASSPATH"=>"#{libexec}/lib/hadoop-common-*.jar", "CFLAGS"=>"-m64", "CXXFLAGS"=>"-m64", "CPPFLAGS"=>"-I/System/Library/Frameworks/JavaVM.framework/Versions/Current/Headers" }, "ant compile-native tar")
```

Comment:
Using `Formula.system` instead of `Kernel.system` doesn't work, it broke the build. I suspect the `ENV` vars aren't exported correctly when `Formula.system` is used.